src/app/pages/MarginTradePage/types.ts

```diff
@@ -2,13 +2,112 @@
 import { TradingPairType } from 'utils/dictionaries/trading-pair-dictionary';
 import { Asset } from 'types/asset';
 import { TradingPosition } from '../../../types/trading-position';
+import { BigNumber } from 'ethers';
 
-export interface MarginTradePageState {
+export const MARGIN_SLIPPAGE_DEFAULT = 0.5;
+
+export interface IMarginTradePageState {
   pairType: TradingPairType;
   collateral: Asset;
   amount: string;
   leverage: number;
   position: TradingPosition;
+  notificationWallet?: string;
+  notificationToken?: string;
+  notificationUser?: NotificationUser;
+}
+
+export type ContainerState = IMarginTradePageState;
```

Comment:
I don't see a reason for that, just delete this line and use `IMarginTradePageState` directly. If you want to rename it in the place of usage, you can do it in `import`.

Reply:
fixed